R/tm_g_scatterplotmatrix.R

```diff
@@ -524,7 +532,8 @@ get_scatterplotmatrix_stats <- function(x, y,
       if (.f_args$method == "spearman") {
         return("rho:-")
       }
+    } else {
+      return("-")
     }
```

Comment:
Can you revert this?

The previous implementation returned `"-"` when both there is no method or it has a name that is different than the options.

Either that or replace the contents of `else` clause with: 

```r
  # ...
  } else {
    checkmate::assert_list(.f_args)
    switch(
      ifelse(is.null(._f_args$method), "", ._f_args$method),
      "pearson" = "cor:-",
      "kendall" = "tau:-",
      "spearman" = "rho:-",
      "-"
    )
  }
}
```

Reply:
😱 I think I am lost in {}.

Reply:
[b9db727](https://github.com/insightsengineering/teal.modules.general/pull/661/commits/b9db72789d12ec0d57bd3b8ee81684c863a2ec8d)